CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ### Changed
 
+- Change the default port to 8150 to avoid port conflicts in the CAPI setup.
```

Comment:
What is actually conflicting with this port?

I am not sure if there is something else (e.g. monitoring) depending on this port

Reply:
It should be scraped via the service https://github.com/giantswarm/chart-operator/blob/master/helm/chart-operator/templates/service.yaml#L10 so I don't think monitoring would be affected. Still might be worth confirming.

We use `hostNetwork` because it needs to run without CNI.